bucketeer/src/test/kotlin/io/bucketeer/sdk/android/internal/remote/ApiClientImplTest.kt

```diff
@@ -177,22 +177,22 @@ internal class ApiClientImplTest {
       apiKey = "api_key_value",
       featureTag = "feature_tag_value",
       moshi = moshi,
-      defaultRequestTimeoutMillis = 1_000,
     )
 
     val (millis, result) = measureTimeMillisWithResult {
       client.getEvaluations(
         user = user1,
         userEvaluationsId = "user_evaluation_id",
+        timeoutMillis = TimeUnit.SECONDS.toMillis(2),
```

Comment:
Because this case is to test using a custom timeout, it should be set here instead of setting the default timeout in the constructor.